lib/sort_by_length.rb

```diff
@@ -1,7 +1,19 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(1)
 def sort_by_length(my_sentence)
```

Comment:
This doesn't actually sort the array, it makes a single pass of BubbleSort.  It may be passing the given tests, but it's not actually doing a complete sort.